Balances/Nicehash.ps1

```diff
@@ -1,4 +1,4 @@
-using module ..\Include.psm1
+﻿using module ..\Include.psm1
```

Comment:
Double check that the encoding is correct.

Reply:
ï»¿Set-Location 

Encoding **IS** UTF-8 with BOM

Reply:
Encoding IS UTF-8 with BOM